lib/p5.js

```diff
@@ -3956,9 +3956,10 @@ amdclean['inputtouch'] = function (require, core) {
   function getTouchPos(canvas, e, i) {
     i = i || 0;
     var rect = canvas.getBoundingClientRect();
+    var touch = e.touches[i] || e.changedTouches[i];
```

Comment:
on 'touchend', e.touches = [] and e.changedTouches = [lastTouch] so the result of this will be ```touch = lastTouch``` which I don't think we want because then the p5 ```touches``` property won't be empty like it should after the last 'touchend'.